nats-native/src/main/java/org/ballerinalang/nats/Constants.java

```diff
@@ -53,12 +52,10 @@
 
     public static final String NATS = "nats";
     public static final String ORG_NAME = "ballerinax";
-    public static final String VERSION = "1.0.7";
 
     // Represents nats package.
-    public static final String NATS_PACKAGE = ORG_NAME + ORG_NAME_SEPARATOR + NATS + ":" + VERSION;
-
-    public static final Module NATS_PACKAGE_ID = new Module(ORG_NAME, "nats", VERSION);
+    public static final String NATS_PACKAGE = ORG_NAME + ORG_NAME_SEPARATOR + NATS + ":"
```

Comment:
Why do we need this?

Reply:
Fixed in c8e94613ed7ff23ec27267c9e3a2bb186a1d285e